summarizer/ai_applications/base/Containerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi9/python-39:1-158
-WORKDIR /locallm
-COPY requirements.txt /locallm/requirements.txt
+WORKDIR /summarizer
+COPY base/requirements.txt .
 RUN pip install --upgrade pip
 RUN pip install --no-cache-dir --upgrade -r /locallm/requirements.txt
```

Comment:
change to /summarizer/requirements.txt

Reply:
the `requirements.txt` is in `summarizer/base/requirement's.txt` the same way as in the other directories. Should it be in a different location?

Reply:
nvm. I see where the error is.